abstract/myst.yml

```diff
@@ -1,52 +1,54 @@
-  # title: TITLE
-  # description: DESCRIPTION
+  id: ubcgif-2020-heagy-etal-seg
+  title: Machine learning for the classification of unexploded ordnance (UXO) from electromagnetic data
+  description: Presenting an approach for using convolutional neural networks to classify unexploded ordnance directly from time-domain electromagnetic data
   authors:
-    # - name: AUTHOR 1
-    #   email: EMAIL
-    #   corresponding: true
-    #   orcid: ORCID
-    #   affiliation:
-    #     id: ubc
-    #     name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
-    # - name: Lindsey J. Heagy
-    #   email: lheagy@eoas.ubc.ca
-    #   corresponding: true
-    #   orcid: 0000-0002-1551-5926
-    #   affiliation: ubc
-    # - name: Douglas W. Oldenburg
-    #   affiliation: ubc
-    #   email: doug@eos.ubc.ca
-    #   orcid: 0000-0002-4327-2124
-  # banner: banner.png
+    - name: Lindsey J. Heagy
+      email: lheagy@eoas.ubc.ca
+      corresponding: true
+      orcid: 0000-0002-1551-5926
+      affiliation:
+        id: ubc
+        name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
+    - name: Douglas W. Oldenburg
+      email: doug@eos.ubc.ca
+      orcid: 0000-0002-4327-2124
+      affiliation: ubc
+    - name: Fernando Pérez
+      affiliation: University of California Berkeley
+    #   email: 
+    #   orcid: 
+    - name: Laurens Beran
+      affiliation: Black Tusk Geophysics
+    #   email: 
+    #   orcid: 
+  banner: banner.png
   thumbnail: thumbnail.png
-  # subject: Electromagnetics
+  subject: Electromagnetics
   venue:
-    # title: CONFERENCE NAME
-    # url: CONFERENCE URL
-  license: CC-BY-4.0
-  open_access: true
-  # doi: DOI
+    title: SEG International Exposition and 90th Annual Meeting
+    url: https://library.seg.org/doi/10.1190/segam2020-3428369.1
+  # license: CC-BY-4.0
+  # open_access: true
+  doi: 10.1190/segam2020-3428369.1
   abbreviations:
-    # DC: direct current
-    # EM: electromagnetics
-  # github:
+    UXO: Unexploded ordnance
+    EM: electromagnetics
+    TDEM: time-domain electromagnetic
+  github: https://github.com/simpeg-research/heagy-et-al-2020-uxo-seg
   # bibliography: []
-  # keywords:
-  #   - Controlled source electromagnetics (CSEM)
-  #   - Downhole methods
-  #   - Numerical modelling
-  #   - Electrical properties
-  #   - Magnetic properties
+  keywords:
+    - Electromagnetics
+    - Machine Learning
+    - Near Surface
 site:
   template: article-theme
   actions:
-    # - title: pdf
-    #   url: ARTICLE-NAME.pdf
-    #   static: true
-    # - title: slides
-    #   url: presentation/SLIDES-NAME.pdf
-    #   static: true
+    - title: pdf
+      url: heagy-et-al-2020-machine-learning-for-the-classification-of-unexploded-ordnance-(uxo)-from-electromagnetic-data.pdf
+      static: true
+    - title: slides
+      url: 2020-Heagy-et-al-UXO.pdf
```

Comment:
this is in the `presentations` folder, so the path should be `presentations/2020-Heagy-et-al-UXO.pdf`